docs/service-introspection.md

```diff
@@ -126,11 +129,16 @@ Usage:
 spk deployment onboard|o [options]`
 
 Options:
-  -n, --storage-account-name <storage-account-name>                Account name for the storage table
-  -l, --storage-location <storage-location>                        Azure location for Storage account and resource group when they do not exist
-  -r, --storage-resource-group-name <storage-resource-group-name>  Name of the resource group for the storage account
-  -k, --key-vault-name <key-vault-name>                            Name of the Azure key vault
-  -h, --help                                                       Usage information
+  -s, --storage-account-name <storage-account-name>                 Azure storage account name
+  -t, --storage-table-name <storage-table-name>"                    Azure storage table name
+  -l, --storage-location <storage-location>                         Azure location to create new storage account when it does not exist
```

Comment:
No, we are not offering the location in the `config` as well as `types.d.ts` since it is only needed once while onboarding the introspection which creates the storage account if necessary. Let me know if I miss any scenarios that requires it to be in config so we can add it.